pkg/restic/restorer.go

```diff
@@ -84,14 +91,15 @@ func newRestorer(
 	return r
 }
 
-func (r *restorer) RestorePodVolumes(restore *velerov1api.Restore, pod *corev1api.Pod, sourceNamespace, backupLocation string, log logrus.FieldLogger) []error {
-	// get volumes to restore from pod's annotations
-	volumesToRestore := GetPodSnapshotAnnotations(pod)
+// TODO(2.0)
```

Comment:
can drop this TODO comment since it's commented at the helper.

Reply:
That was not meant to be there!